packages/x-components/src/utils/normalize.ts

```diff
@@ -8,9 +8,10 @@
  * @returns If the environment supports string normalization, the string without the combining
  * diacritical marks. Else the same string.
  */
-const removeCombiningDiacriticalMarks = String.prototype.normalize
-  ? (string: string) => string.normalize('NFD').replace(/[\u0300-\u036f]/g, '')
-  : (string: string) => string;
+const removeCombiningDiacriticalMarks =
+  typeof String.prototype.normalize === 'function'
```

Comment:
TypeScript was complaining that the normalize function is always available, so the condition would always return true. This happens for modern browsers, but not for IE or android webviews. Changing the assertion made TS stop complaining, and now the assertion is also safer.